streams/core.py

```diff
@@ -524,17 +530,23 @@ def update(self, x, who=None):
 
 
 class combine_latest(Stream):
-    def __init__(self, *children):
+    def __init__(self, *children, emit_on=None):
```

Comment:
I recall that earlier you used numbers here to refer to the children by index location.  I can imagine that also being useful.  Perhaps we can accept these as well and normalize to the streams?

```python
emit_on = tuple(children[x] if isinstance(x, int) else x for x in emit_on)
```